Misc/NEWS

```diff
@@ -10,6 +10,9 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30721: ``print`` now shows correct usage hint for using Python 2
+  redirection syntax. Patch by Sanyam Khurana.
+
```

Comment:
We don't edit `Misc/NEWS` directly any more. Instead, you'll want to add a blurb entry under https://github.com/python/cpython/tree/master/Misc/NEWS.d/next/Core%20and%20Builtins

See the existing entries and https://docs.python.org/devguide/committing.html#what-s-new-and-news-entries for more details